configs/nextflow.hopper.config

```diff
@@ -302,4 +302,48 @@ myeloid {
     params.gens_accessdir = "/access/myeloid_const/plot_data"
     params.accessdir = "/access/myeloid_const/"
   } 
+
+modycf {
+    params.outdir = "${params.resultsdir}${params.dev_suffix}"
+    params.subdir = 'modycf'
+    params.crondir = "${params.outdir}/cron/"
+    params.intersect_bed = "/fs1/resources/ref/hg38/bed/twist-modycf/MODY_CF_updated.bed"
+    params.align = true
+    params.varcall = true
+    params.annotate = true
+    params.gatkcov = false
+    params.exome = false
+    params.sv = true
+    params.gatkcnv = true
+    params.str = false
+    params.onco = false
+    params.umi = true
+    params.delly = false
+    params.distributed = false
+    params.intervals = "/fs1/resources/ref/hg38/bed/twist-modycf/MODY_CF_updated.interval_list"
+    params.bedgz = "/fs1/resources/ref/hg38/bed/twist-modycf/MODY_CF_updated.bed.gz"
+    params.panelhs = "sentieon driver -r ${params.genome_file} -t ${params.cpu_all} -i "
```

Comment:
This confuses me. Why is there code in the config?

Reply:
this pipeline had a sole purpose of doing WGS at the start. For wgs we run wgs_metrics but for panels we need to run hs_metrics. This is really old legacy code when I didnt know of params. We could clean this up, but also it works so really no hurry in this PR

Reply:
Yes, OK, I understand. I was guessing it was something like that. I think this kind of thing would be nice to clean up long term. I agree that there is no need to fix it right here and now. But if we see a long-term future of this future, we could start thinking about how to over time do a gradual cleanup.

Reply:
I opened an issue #83 where I dump future cleanup tasks